.github/workflows/build.yml

```diff
@@ -78,6 +78,19 @@ jobs:
       - run: ${{ matrix.task }}
         continue-on-error: ${{ matrix.experimental }}
 
+  check:
```

Comment:
"check" is vague (people might think it means a linkcheck, e.g.), so it's a little unnerving to see it be skipped in the Actions tab. Could we be more specific? I don't know the conventions for names, but if length and hyphens are OK, one possibility is something like sdk-files-are-up-to-date.